src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsole.cs

```diff
@@ -621,11 +631,12 @@ private void WriteProgress(string operation, int percentComplete)
 
             if (percentComplete == 100)
             {
-                HideProgress();
+                await HideProgressAsync();
             }
             else
             {
-                VsStatusBar.Progress(
+                var vsStatusBar = await GetVsStatusBarAsync();
```

Comment:
It's probably only necessary to switch here, not at the top.

Reply:
Ok. Addressed.